src/Agent.Worker/Build/FileContainerServer.cs

```diff
@@ -124,15 +124,19 @@ public class FileContainerServer
             }
         }
 
-        private async Task<List<string>> ParallelUploadAsync(IAsyncCommandContext context, List<string> files, int concurrentUploads, CancellationToken token)
+        private async Task<UploadResult> ParallelUploadAsync(IAsyncCommandContext context, IReadOnlyList<string> files, int concurrentUploads, CancellationToken token)
         {
-            // return files that fail to upload
-            List<string> failedFiles = new List<string>();
+            // return files that fail to upload and total artifact size
+            var uploadResult = new UploadResult
```

Comment:
add a default constructor to this, so only do `var uploadResult = new UploadResult();`

Reply:
Added a default constructor